app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,200 @@
  *= require_tree .
  *= require_self
  */
+
+:root {
+  --main-color: #3778c2;
+  --second-color: #5fb523;
+  --text-color: #434b54;
+  --white-color: #fff;
+  --background-color: #f5f6f8;
+  --bold-weight: 800;
+  --light-weight: 300;
+}
+
+body {
+  font-family: 'Proxima Nova', sans-serif;
+  color: var(--text-color);
+  font-size: 10px;
+  background-color: var(--background-color);
+}
+
+.container {
+  box-sizing: border-box;
+  margin: 10px 20px;
+}
+
+.card {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  margin-bottom: 10px;
+}
+
+.card .info {
+  border: 3px solid black;
+  width: 70vw;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card .info p {
+  align-self: flex-end;
+}
+
+.card .img {
+  width: 30vw;
+  text-align: center;
+}
+
+.posts {
+  display: flex;
+  flex-direction: column;
+}
+
+input[type=submit],
+button {
+  background-color: var(--second-color);
```

Comment:
I'm impressed with your app, it app functions great! 👏  However, it will be a good idea if we can add some few tweaks to the design aspect so as to improve the look and feel. Please take note that it's one of the project's objective that our design closely mimics the given [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=). Let's improve upon the design aspect by introducing the following tweaks:
- [ ] Let's take note that the splash screen actually comes before we login or signup, let's improve upon this aspect as can be seen from the original design.
- [ ] Let's ensure that the green button is is centered on the page, always sticky to the bottom for category and transactions pages, and dimensions similar to the original design (for mobile screens). 
- [ ] Kindly add a back button/arrow within the _nav_ area so as to enable users go back the previous step (mobile screen)
- [ ] The layout within the contents page is quite squashed in. Let's improve this sections by ensuring the contents stretches proportionally across the screen, with same width as the _nav_ section


**Your design:**
**Your splash screen:**        |  **Your categories page:**        |  **Your transaction page**     
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/229333205-f3f943ba-7043-4de8-8288-231c1f59691c.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/229332770-7d82ffb4-217d-4504-857a-c339bedcc509.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/229333093-4594db38-92de-49f8-bed3-01ff0e114115.png">


**Expected design:**
**Original splash screen:**        |  **Original login screen:**        |  **The original transaction:**     
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/198862103-fe9fe8a8-7a06-4a03-92f6-8528c3d4d2f1.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/229333537-cf3d2e79-4c88-4eb4-b7b8-00ee40609047.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/227751375-40282971-5bde-49ca-9612-8cc1be58ba99.png">